test/e2e/tests/test_event_generator/test_network_activity.py

```diff
@@ -43,7 +43,7 @@ def test_network_activity(sinsp, run_containers: dict):
         },
         {
             "container.id": generator_id,
-            "evt.args": SinspField.regex_field(fr'^res=0 tuple={ipv4_regex}->10\.2\.3\.4:8192 $'),
+            "evt.args": SinspField.regex_field(fr'^res=0 tuple={ipv4_regex}->10\.2\.3\.4:8192 fd=3(<4>)$'),
```

Comment:
@Molter73 do you know what should i add here? :D 
`fd=3(<4>)` is exactly the same param that the previous event is receiving (the `connect` enter event)...

Reply:
Uh i see, i need to add the full tuple because sinsp adds it :)